### ISSUE TITLE
Image tag details : NUllPointerException 

### ISSUE THREAD

**Issue by pieval** [mkuchin/docker-registry-web#42]
I'm running a NullPOinterException when I try to see image tags details:
```
Cannot invoke method last() on null object. Stacktrace follows:
java.lang.NullPointerException: Cannot invoke method last() on null object
	at docker.registry.web.RepositoryController$_tag_closure10.doCall(RepositoryController.groovy:111)
	at docker.registry.web.RepositoryController.tag(RepositoryController.groovy:109)
	at grails.plugin.cache.web.filter.PageFragmentCachingFilter.doFilter(PageFragmentCachingFilter.java:198)
	at grails.plugin.cache.web.filter.AbstractFilter.doFilter(AbstractFilter.java:63)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
```
Error is located here https://github.com/mkuchin/docker-registry-web/blob/v0.0.4/grails-app/controllers/docker/registry/web/RepositoryController.groovy#L111
In fact problem is that in my image manifest json export I can see history element for which ```container_config.Cmd``` is null ( and also all fields in container_config are empty strings or null objects)
Do you have to manage this cases or is it a bug with my manifests files ?

For information I'm running registry v2.4.1 and for now docker 1.9

**Comment by mkuchin** [comment]
Thanks for bug report. The reason for null cmd may be combination of old docker version 1.9 and new registry version. Can you share json output of such request substituting `$URL`, `$REPO` and `$TAG` with your values:
    
    curl -H "Accept: application/vnd.docker.distribution.manifest.v2+json" http://$URL/v2/$REPO/manifests/$TAG

**Comment by mkuchin** [comment]
Fix released in [version 0.1.2](https://github.com/mkuchin/docker-registry-web/releases/tag/v0.1.2)
